swiper.el

```diff
@@ -733,53 +733,36 @@ When capture groups are present in the input, print them instead of lines."
   (when (bound-and-true-p evil-mode)
     (evil-set-jump)))
 
-(declare-function char-fold-to-regexp "char-fold")
+(defun swiper--normalize-regex (re)
+  "Normalize the swiper regex RE.
+Add a space after a leading `^' if needed and apply
+`search-default-mode' if bound."
+  (replace-regexp-in-string
+   "^\\(\\(?:\\\\(\\)?\\^\\)"
+   (concat "\\1" (if (eq 'swiper (ivy-state-caller ivy-last)) " " ""))
+   (if (fboundp (bound-and-true-p search-default-mode))
```

Comment:
Since you are later passing `search-default-mode` to `funcall`, the correct check is `functionp`, not `fboundp`.

Reply:
Good catch